src/templates/static/legal/tac_sections/tac.jsx

```diff
@@ -9,11 +9,10 @@ const Tac = () => (
         <h2 data-anchor='interpretation'>A. {it.L('Interpretation')}</h2>
         <p>
             {it.L('Depending on the jurisdiction to which a Client\'s account has been attached in accordance with the account opening procedures of the website and as detailed in the About Us section of the website and the type of products offered, the term "the Company" shall denote either')}&nbsp;
-            <span data-show='-eucountry'>{it.L('Binary (V) Ltd.,')}&nbsp;</span>
             <span data-show='-eucountry'>{it.L('Binary (C.R.) S.A.,')}&nbsp;</span>
             <span data-show='-eucountry'>{it.L('Binary (BVI) Ltd.,')}&nbsp;</span>
```

Comment:
you can remove Binary (BVI) Ltd. as it is not allowed to offer binary options

Reply:
sure